src/Refactoring-UI/ReRemoveDuplicateMethodsChoice.class.st

```diff
@@ -0,0 +1,19 @@
+Class {
+	#name : 'ReRemoveDuplicateMethodsChoice',
+	#superclass : 'ReRemoveMethodChoice',
+	#category : 'Refactoring-UI-Choices',
+	#package : 'Refactoring-UI',
+	#tag : 'Choices'
+}
+
+{ #category : 'execution' }
+ReRemoveDuplicateMethodsChoice >> action [
+
+	driver removeDuplicateMethods
+]
```

Comment:
This doesn't feel like a choice, but an additional option that can be done besides pull up. This doesn't have to be resolved in this PR, but I see these as checkboxes that user can tick and we do only those that user selected.

We can create an issue for this and resolve it in a separate PR.


These can be choices as well, but we have to combine them with other so we have 4 options and need to create at least 8 choice entries to cover all the cases, but I do not recommend this approach.

Reply:
I have currently implemented it using a checkbox as shown in this video. I am not sure this is what you meant in your comment. If not, I will create an issue and we can continue this discussion there,
![RemoveDuplicateMethods](https://github.com/user-attachments/assets/4db72a2d-d9e7-4698-baeb-ac5e3641cea0)

Reply:
I don't see a checkbox in the video, I will have to load the code and test it.

Reply:
@balsa-sarenac Yes please, I would like to know what you think. Now that this PR is merged, you can make the checkbox appear by pulling up the method `MyClassBeta>>methodDuplicatedInSubclass`